pyfolio/tears.py

```diff
@@ -64,8 +64,14 @@ def create_returns_tear_sheet(returns, live_start_date=None,
         Daily non-cumulative returns of the first benchmark.
     return_fig : boolean, optional
         If True, returns the figure that was plotted on.
+    set_context : boolean, optional
+        If True, set default plotting style context.
     """
 
+    if set_context:
```

Comment:
Is there a reason we're not using a proper `with` block here?  Elsewhere I've used the following idiom for conditional contextmanagers:

```
from contextlib import contextmanager

@contextmanager
def nullctx():
    "No-op contextmanager"
    yield


# Inside some function that maybe wants to use a context manager:
with some_context_manager if some_expression else nullctx():
    # Do context things.
```